### PR TITLE
Feature/qol dec22

### PR DESCRIPTION
**Pull request by BrychanOdlum**

### CODE REVIEW

File: src/app/components/Core/Layout/Sidebar/SidebarLibraryMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  return connectDropTarget(
+    <div
+      className={cx(
+        classes.menu,
+        { [classes.dndSeeking]: activeDndItem },
+        { [classes.dndHovered]: isDndOver }
+      )}
+    >
+      <h3>{title}</h3>
+      <ul>
```
[the remark this answers is not in the document]
Comment (BrychanOdlum) [reply src/app/components/Core/Layout/Sidebar/SidebarLibraryMenu.jsx]: @raphaelvigee prettier conflicting with ESLint here :'(

File: src/app/assets/Styles/contextMenu.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.react-contextmenu-item {
+  background-color: transparent;
+
+  &:hover {
+    background-color: #fe2851;
+    color: #ffffff;
+  }
+}
```
Comment (raphaelvigee) [inline src/app/assets/Styles/contextMenu.scss]: \n

File: src/app/assets/Styles/toaster.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  color: #27AE60;
+}
+.s-alert-warning {
+  background: #fff;
+  color: #F1C40F;
+}
+.s-alert-error {
+  background: #fff;
+  color: #E74C3C;
+}
```
Comment (raphaelvigee) [inline src/app/assets/Styles/toaster.scss]: \n

File: src/app/assets/Styles/toaster.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+.s-alert-close::before,
+.s-alert-close::after {
+  background: #666666;
+}
+
+.s-alert-info {
+  background: #fff;
+  color: #00A2D3;
+}
+.s-alert-success {
```
Comment (raphaelvigee) [inline src/app/assets/Styles/toaster.scss]: \n

File: src/app/assets/Styles/toaster.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  background: #fff;
+  color: #27AE60;
+}
+.s-alert-warning {
```
Comment (raphaelvigee) [inline src/app/assets/Styles/toaster.scss]: \n

File: src/app/assets/Styles/toaster.scss [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  background: #fff;
+  color: #F1C40F;
+}
+.s-alert-error {
```
Comment (raphaelvigee) [inline src/app/assets/Styles/toaster.scss]: \n

File: src/app/components/Core/Artists/ArtistPage.jsx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+            ))}
+          </div>
+        )}
+      </PageContent>
+    );
+  }
+}
+
+ArtistPage.propTypes = {
+  match: PropTypes.any,
```
Comment (raphaelvigee) [inline src/app/components/Core/Artists/ArtistPage.jsx]: isRequired

File: src/app/components/Core/Artists/ArtistPage.jsx [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+};
+
+ArtistPage.defaultProps = {
+  match: null,
```
Comment (raphaelvigee) [inline src/app/components/Core/Artists/ArtistPage.jsx]: Remove

File: src/app/components/Core/Artists/ArtistPage.scss [hunk 9]
```diff
@@ -0,0 +1,8 @@
+.albumsGrid {
+  display: flex;
+  flex-wrap: wrap;
+
+  > div {
+    margin: 15px 10px;
+  }
+}
```
Comment (raphaelvigee) [inline src/app/components/Core/Artists/ArtistPage.scss]: \n

File: src/app/components/Core/Layout/Sidebar/Sidebar.scss [hunk 10]
```diff
@@ -21,6 +21,15 @@
       background: $main_background_color;
       padding: 10px 15px;
       border-radius: 5px;
+      border: solid 1px transparent;
+
+      &.dndSeeking {
+        border: dashed 1px lighten($primary_color, 20%);
+      }
+      &.dndHovered {
```
Comment (raphaelvigee) [inline src/app/components/Core/Layout/Sidebar/Sidebar.scss]: \n

File: src/app/components/Core/Layout/Sidebar/SidebarLibraryMenu.jsx [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+          <MenuItem to={item.to} label={item.label} />
+        ))}
+      </ul>
+    </div>
+  );
+}
+
+SidebarLibraryMenu.propTypes = {
+  title: PropTypes.string.isRequired,
+  items: PropTypes.any.isRequired,
```
Comment (raphaelvigee) [inline src/app/components/Core/Layout/Sidebar/SidebarLibraryMenu.jsx]: array

File: src/app/components/Core/Layout/Sidebar/SidebarMenu.jsx [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+          <MenuItem to={item.to} label={item.label} />
+        ))}
+      </ul>
+    </div>
+  );
+}
+
+SidebarMenu.propTypes = {
+  title: PropTypes.string.isRequired,
+  items: PropTypes.any.isRequired,
```
Comment (raphaelvigee) [inline src/app/components/Core/Layout/Sidebar/SidebarMenu.jsx]: array

File: src/app/components/Core/Radio/RadioPage.jsx [hunk 13]
```diff
@@ -4,7 +4,12 @@ import classes from './RadioPage.scss';
 export default function RadioPage() {
   return (
     <div className={classes.radioContainer}>
-      <span className={classes.comingSoon}>Coming Soon!</span>
+      <div className={classes.comingSoonIcon}>
+        <i className={'fas fa-broadcast-tower'} />
+      </div>
+      <span className={classes.comingSoon}>
+        {"We're working on bringing Radio to you ASAP. Check back later."}
```
Comment (raphaelvigee) [inline src/app/components/Core/Radio/RadioPage.jsx]:
```
`
```

File: src/app/components/Core/Albums/AlbumPanel.scss [hunk 14]
```diff
@@ -39,6 +39,17 @@
       display: block;
       color: $tertiary_color;
       margin-bottom: 10px;
+      > a:hover {
+        color: #fe2851;
+      }
+      .musicLink {
```
Comment (raphaelvigee) [inline src/app/components/Core/Albums/AlbumPanel.scss]: \n

File: src/app/components/Core/Albums/AlbumPanel.scss [hunk 15]
```diff
@@ -39,6 +39,17 @@
...
+        margin-left: 15px;
+        font-size: 0.9em;
+        color: #999;
+        &:hover {
```
Comment (raphaelvigee) [inline src/app/components/Core/Albums/AlbumPanel.scss]: \n

**Pull request merged**